src/engine/SCons/Tool/msvsTests.py

```diff
@@ -623,50 +662,69 @@ def test_config_generation(self):
                         'debug=False target_arch=32',
                         'debug=True target_arch=x64', 
                         'debug=False target_arch=x64']
-        
-        # Tuple list :   (parameter,        dictionary of expected result per variant)
-        tests_cmdargs = [(None,            dict.fromkeys(list_variant, '')), 
-                         ('',              dict.fromkeys(list_variant, '')), 
-                         (list_cmdargs[0], dict.fromkeys(list_variant, list_cmdargs[0])),
-                         (list_cmdargs,    dict(list(zip(list_variant, list_cmdargs))))]
-        
+        list_cppdefines = ['_A', '_B', 'C', None]
+        list_cpppaths = [r'C:\test1', r'C:\test1;C:\test2', Dir('subdir'), None]
```

Comment:
What are you trying to do here? get the current test dir?

Reply:
This is testing proper handling and conversion of the cpppaths and cppdefines parameters, similar to the existing cmd args parameter tests. These are dummy directories for test only.

Reply:
Why use Dir() then? and not 'subdir' ?

Reply:
Because I wanted a test case that accounts for consumers passing in a Dir object in the cpppaths parameter. Yesterday I ran into pickling problems with Dir objects and fixed them in the msvs.py, so I wanted one of the permutations of the test cases to cover that use case.